FanartHandler/FanartSelectedOther.cs

```diff
@@ -13,6 +13,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using WindowPlugins.GUITVSeries;

```

Comment:
When we move the analysis to the TVSeries Helper, this dependency will not be needed.